target/linux/ramips/dts/mt7628an.dtsi

```diff
@@ -457,10 +468,8 @@
 		interrupt-parent = <&cpuintc>;
 		interrupts = <4>;
 
-		resets = <&rstctrl 26 &rstctrl 27>;
+		resets = <&sysc 26>, <&sysc 27>;
```

Comment:
I think you can remove <&sysc 27> from here. Only 26 makes sense.

Reply:
Thanks, you are right. MT7620 and MT7628 only have one PCIe port. I have removed it from my local staging tree. I'll push it on the weekend.

By the way, `usbphy` nodes in all SoC dtsi also have similar issues. The datasheet shows that they only have one reset line, but the driver has defined two. Perhaps we can use a patch to remove the extra one.